packages/create-sitecore-jss/src/templates/node-headless-ssr-experience-edge/src/config.ts

```diff
@@ -31,12 +37,10 @@ const config = {
   /**
    * Port which will be used when start sample
    */
-  port: process.env.PORT || 3000,
+  port: (process.env.PORT as number | undefined) || 3000,
```

Comment:
is it required to provide `undefined` type? It should take `3000` in case if it will be `undefined`.
I see that on the line 29 you don't case anything, so can we just leave `PORT` as it is?